lib/core/services/api/api_service_impl.dart

```diff
@@ -61,4 +63,24 @@ class ApiServiceImpl extends ApiService {
       rethrow;
     }
   }
+
+  @override
+  Future<void> getLoginRegister(String name, String email, String password) async {
+    final body ={
+      'name': name,
+      'email': email,
+      'password': password
+    };
+    try {
+      final response = await dio.post('/auth/register', data: body );
+      if(response == 200 && response != null){
+        print('success');
```

Comment:
After registration is successful, handle the response user data from API, and save to preferences for later use of the app.